### DISCUSSION TITLE
How to enable Slack messages for Critical Sigma alerts? License?

### DISCUSSION THREAD

**Discussion by innovate-support** [Security-Onion-Solutions/securityonion#13099] [Unsupported Versions]
I've recently set up an SO server specifically intended to gather Elastic Agent data from remote employee Windows laptops.  I updated it to v2.4.70 and it's Alerting correctly when I run test Malware.  I would like to additionally have it send a Slack message when it sees any enabled 'Sigma' alert that is deemed 'Critical' severity.  I've been reading through the docs found at
https://docs.securityonion.net/en/latest/notifications.html#slack
https://docs.securityonion.net/en/latest/sigma.html#sigma

Per the instructions I added my slack_webhook_url in 'Administration> Configuration> elastalert> Alerter Parameters' and then went to the new Detections section and ran 'Full Update' on ElastAlert.  I saw the note in the 'Alerting Parameters' section stating that it 'Requires a valid Security Onion license key'.

I've looked on the SO website and I'm not seeing anywhere to purchase a license.  Where/how do I purchase and apply it, and am I correct in assuming that it unlocks a currently hidden 'Edit' capability?  And also, will I have to manually edit all 87 'Enabled>Sigma>Critical' rules and define each one to notify via Slack?

**Comment by TOoSmOotH** [comment]
Please see: https://github.com/Security-Onion-Solutions/securityonion/discussions/13105

Previous versions  of 2.4 only supported SMTP notifications.

**Reply by innovate-support** [reply]
I don't understand how this is an answer to my question.

**Reply by TOoSmOotH** [reply]
You can email pro@securityonion.com to get info about how to purchase pro.

**Reply by innovate-support** [reply]
That email bounced back as non-existent.

**Reply by dougburks** [reply]
The alias hasn't propagated to the securityonion.com domain yet. Please try pro@securityonionsolutions.com. Thanks!
